src/internal/m365/collection/drive/item_handler.go

```diff
@@ -21,16 +22,68 @@ import (
 // backup
 // ---------------------------------------------------------------------------
 
+type baseItemHandler struct {
```

Comment:
This name (along with the other "itemFoo" names) is out of date, and was originally chosen to represent the file name it came from.  Would you mind making this change as long as you're in the refactor?
```suggestion
type baseUserDriveHandler struct {
```

Reply:
I'll spin up another PR for this and other naming changes